src/webgpu/shader/execution/expression/call/builtin/quadBroadcast.spec.ts

```diff
@@ -447,6 +447,9 @@ predication filters are skipped.
     const wgsl = `
 enable subgroups;
 
+diagnostic(off, subgroup_uniformity);
+diagnostic(off, subgroup_branching);
```

Comment:
per agreement today, remove subgroup_branching.
But shouldn't be a blocker since it should be a warning if it doesn't exist.